semantic/semantic.cabal

```diff
@@ -42,6 +42,10 @@ common haskell
     -DBAZEL_BUILD=0
   if (impl(ghc >= 8.8))
     ghc-options: -Wno-missing-deriving-strategies
+  if (impl(ghc >= 8.10))
+    ghc-options:
+      -Wno-missing-safe-haskell-mode
+      -Wno-prepositive-qualified-module
```

Comment:
I’ve also (I think) got everything building with `ghc` 8.10, tho I haven’t e.g. made CI use that yet.